packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -742,7 +742,7 @@ export class PerformanceEvent {
 			this.reportEvent("start");
 		}
 
-		if (typeof window === "object" && window?.performance?.mark) {
+		if (typeof window === "object" && window?.performance?.mark !== undefined) {
```

Comment:
Nothing that this would technically change behavior if `window?.performance?.mark` was ever `null` but I don't think that would ever happen.

Reply:
I'll check both to be safe. APIs like `window` are pretty old and older APIs are more likely to use null in my experience.